vdb/lib/aqua.py

```diff
@@ -110,13 +110,15 @@ def is_supported_source(zfname):
             "oval",
             "glad",
             "mariner",
+            f"cvrf{os.sep}suse{os.sep}suse",
```

Comment:
This removes data belonging suse enterprise version leaving only the opensuse ones, which is probably ok for us.